source/_integrations/shelly.markdown

```diff
@@ -20,7 +20,13 @@ To add a Shelly device to your installation, go to **Configuration** >> **Integr
 
 A device can also be added manually, click the button with `+` sign on the integrations page and from the list of integrations, select **Shelly** and follow the instructions shown.
 
+<div class="note">
+You need to configure and connect the device to same network using the Shelly cloud App or device Web page.
+Integration is communicating directly with the device; cloud connection is not needed.
```

Comment:
To the "same network" of what?

Reply:
I followed the previous definition which shows in the beginning of the documentation:  "Devices will be automatically discovered when on the same network",
Shelly devices and Home assistant needs to be on the same network so that CoAP messages are able to pass from Shelly devices to Home Assistant.

Is the following more clear:
"You need to configure and connect the device to same network of Home Assistant using the Shelly cloud App or device Web page."

Reply:
Wait, so it is already written above? Why add a note?

Reply:
I added the note based on feedback from users:
Users did not clearly understand that they need to first configure the device and connect to WIFI before trying to add them to HA.
The first sentence actually talks about auto discovery, while it is also possible to work using difference networks which are accessible from each other without being on the same network.

The main idea is to tell the user that the devices needs to be connected to network. Using the phrase "same network" may confuse some users, that is why I avoided it and I assume it is the same reason it is not written above.

Can you suggest a more clear way to emphasize this requirement?
If it is not clear I will remove it and leave only the cloud connection note.

Reply:
I think we should avoid creating big notices/info/warning boxes in general, unless really important. 

IMHO, we can just add a line to the initial part in normal text or incorporate it into the existing text.

For example, rewriting the first section to:

> To add a Shelly device to your installation, make sure they are connected to your Wi-Fi network first. Next, go to **Configuration** >> **Integrations** in the UI.

> If the new devices are on the same network as Home Assistnat, they will be discovered automatically. Clicking "Configure" on the discovered device adds it to Home Assistant. If your device isn't discovered automatically, click the + to add a new integration and search for Shelly to add it manually.

Reply:
I rewrite the first section similar to what you suggested (I keep some of the original text).

I also keep the note **"Integration is communicating directly with the device; cloud connection is not needed."** 
However this can also be added to the normal text if you think it is too "big"